src/index.js

```diff
@@ -72,6 +74,7 @@ const deleteTodo = () => {
         });
         todosArray = todos;
         localStorage.setItem('todos', JSON.stringify(todos));
+        console.log(todos);
         // Reload the page
```

Comment:
- [ ] Kindly remove the console.log statement from your application as it is unprofessional to have data being logged to the console when the application is in production. It impacts negatively the amount on the performance of the application and is based for user interaction.